linera-views/src/scylla_db.rs

```diff
@@ -391,6 +374,11 @@ impl ScyllaDbClientInternal {
         }
     }
 
+    fn contains_forbidden_characters(table_name: &str) -> bool {
+        let forbidden_chars = ".-!@#$%^&*()";
+        table_name.chars().any(|c| forbidden_chars.contains(c))
+    }
```

Comment:
My reading of [the documentation](https://opensource.docs.scylladb.com/stable/cql/ddl.html#common-definitions) would be:

```suggestion
    fn is_allowed_name(table_name: &str) -> bool {
        !table_name.is_empty()
            && table_name.len() <= 48
            && table_name.chars().all(|c| c.is_ascii_alphanumeric() || c == '_')
    }
```